test/build.info

```diff
@@ -178,6 +178,14 @@ IF[{- !$disabled{tests} -}]
   INCLUDE[tls13ccstest]=../include ../apps/include
   DEPEND[tls13ccstest]=../libcrypto ../libssl libtestutil.a
 
+  SOURCE[tls12groupselection_test]=tls12groupselection_test.c helpers/ssltestlib.c
+  INCLUDE[tls12groupselection_test]=../include ../apps/include
+  DEPEND[tls12groupselection_test]=../libcrypto ../libssl libtestutil.a
+
+  SOURCE[tls13groupselection_test]=tls13groupselection_test.c helpers/ssltestlib.c
+  INCLUDE[tls13groupselection_test]=../include ../apps/include
+  DEPEND[tls13groupselection_test]=../libcrypto ../libssl libtestutil.a
```

Comment:
Exclude builds of these if no ecx or TLSv1.2/TLSv1.3 (as appropriate) are disabled.